Revert a patch from upstream:

author	Neil Brown <neilb@suse.de>
Fri, 16 Mar 2007 00:56:59 +0000 (11:56 +1100)
committer	Neil Brown <neilb@suse.de>
Fri, 16 Mar 2007 00:56:59 +0000 (11:56 +1100)
commit	8ab0b89f09dff407101a6f22265873a283e73901
Install mount.nfs* in /sbin instead of /usr/sbin
This patch is now hard-coded in the Makefile.am and cannot
be changed by configure. But as it needs to match what
util-linux does, that is probably a good thing.

In this way we can set at configure where to install sbin tools.

Signed-off-by: Andrei Gherzan <andrei.gherzan@windriver.com>

Index: nfs-utils-1.2.3/utils/mount/Makefile.am
===================================================================
--- nfs-utils-1.2.3.orig/utils/mount/Makefile.am	2012-06-11 11:58:01.079309137 +0300
+++ nfs-utils-1.2.3/utils/mount/Makefile.am	2012-06-11 11:59:02.915306925 +0300
@@ -1,9 +1,5 @@
 ## Process this file with automake to produce Makefile.in
 
-# These binaries go in /sbin (not /usr/sbin), and that cannot be
-# overriden at config time.
-sbindir = /sbin
-
 man8_MANS	= mount.nfs.man umount.nfs.man
 man5_MANS	= nfs.man
 
@@ -31,9 +27,9 @@
 
 install-exec-hook:
 	(cd $(DESTDIR)$(sbindir) && \
-	  ln -sf mount.nfs mount.nfs4 && \
-	  ln -sf mount.nfs umount.nfs && \
-	  ln -sf mount.nfs umount.nfs4 && \
+	  ln -sf $(sbin_PROGRAMS) mount.nfs4 && \
+	  ln -sf $(sbin_PROGRAMS) umount.nfs && \
+	  ln -sf $(sbin_PROGRAMS) umount.nfs4 && \
 	  chmod 4511 mount.nfs )
 uninstall-hook:
 	(cd $(DESTDIR)$(sbindir) && \
